tests/kafka_consumer_manager/test_utils.py

```diff
@@ -25,9 +28,88 @@
 Message = namedtuple("Message", ["partition", "offset", "key", "value"])
 
 
+class TestKafkaAdminGroupReader(object):
+
+    @contextlib.contextmanager
```

Comment:
Writing this as a `pytest.fixture` seems preferable, as tests will only have to do one of these options:
```
@pytest.mark.usefixtures("mock_admin_client")
def test(...)
```
or
```
def test_something(self, mock_admin_client):
```
to get the context manager used. This is easier than adding a context manager explicitly in code, and removes one level of indentation!

Since we never use the value yielded by the context manager, it would be better to use the first option.

Reply:
Other tests were using contextlib, so I figured it would be good to stay consistent, but heh, let's make the world better by using fixtures :) Thanks!